usersync/cookie.go

```diff
@@ -162,9 +215,31 @@ func (cookie *PBSCookie) GetId(bidderName openrtb_ext.BidderName) (id string, ex
 // SetCookieOnResponse is a shortcut for "ToHTTPCookie(); cookie.setDomain(domain); setCookie(w, cookie)"
 func (cookie *PBSCookie) SetCookieOnResponse(w http.ResponseWriter, domain string, ttl time.Duration) {
 	httpCookie := cookie.ToHTTPCookie(ttl)
+	var MAX_COOKIE_SIZE int = 1 << 15 // 32 KB
```

Comment:
We will want this as a config parameter rather than hardcoded. Also a check for a 0 "size" to disable enforcement of a maximum size may not be bad.

Reply:
Do you mean get it from the `configmap.yaml` file itself? Bring it all the way here straight from the `Configuration struct` found in `config/config.go`?

Reply:
What do you mean check for a 0 size? A 0 size `httpCookie.String()`?

Reply:
Yes, bring the config struct, or the relevant value, all the way down to here. It is the configured value for `MAX_COOKIE_SIZE` you should check for a 0 value, as a method of turning off this feature. If a host limits the number of bidders they support, they can limit the size the cookie can grow to.